sys/fs/fuse/fuse_vnops.c

```diff
@@ -2916,7 +2922,7 @@ fuse_vnop_listextattr(struct vop_listextattr_args *ap)
 	if (err != 0) {
 		if (err == ENOSYS) {
 			fsess_set_notimpl(mp, FUSE_LISTXATTR);
-			err = EOPNOTSUPP;
+			err = EXTERROR(EOPNOTSUPP, "This daemon does not implement extattrs");
```

Comment:
Here and elsewhere you repeat the same constant string.  Would it make sense to create a constant for it?
```suggestion
			err = EXTERROR(EOPNOTSUPP, "This server does not implement extended attributes");
```